sonic-net/sonic-buildimage
# Feature added for Wistron 6512-32R platform

#### Why I did it
- Fix transceiver EEPROM and status unstable issue
- Correct temp_max of coretemp
- Support transceiver frequency and TX-power control
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
- Function implementation, code review and functionn test.
#### How to verify it
- Insert and remove transceiver module to check transceiver status
- Check coretemp temp_max
- Using CLI command to get/set transceiver frequency and TX-power


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
